extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/linkHandler/PeertubeStreamLinkHandlerFactory.java

```diff
@@ -27,7 +28,7 @@ public String getUrl(String id) {
 
     @Override
     public String getUrl(String id, String baseUrl) {
-        return baseUrl + VIDEO_ENDPOINT + id;
+        return baseUrl + VIDEO_PATH + id;
     }
 
     @Override
```

Comment:
Please also handle the API URLs in `getId()` here

Reply:
Oh right, for some reason I thought they were already handled in some strange way

Reply:
Those are already handled by the regex: `/videos/(watch/)?([^/?&#]*)`
Both "framatube.org/api/v1/videos/id" and "framatube.org/videos/id" match it and the second group is obtained correctly